eth/backend.go

```diff
@@ -199,8 +200,11 @@ func New(stack *node.Node, config *ethconfig.Config, l1Client sync_service.EthCl
 		return nil, err
 	}
 	if config.CheckCircuitCapacity {
-		tracer := tracing.NewTracerWrapper()
-		eth.blockchain.Validator().SetupTracerAndCircuitCapacityChecker(tracer)
+		eth.asyncChecker = ccc.NewAsyncChecker(eth.blockchain, runtime.GOMAXPROCS(0))
+		eth.asyncChecker.WithOnFailingBlock(func(b *types.Block, err error) {
+			log.Warn("block failed CCC check, it will most likely be reorged by the sequencer", "hash", b.Hash(), "err", err)
```

Comment:
Why "most likely"? Is it possible that it's not reorged?

Reply:
nope, it is just bad wording. will change.